pkgs/top-level/all-packages.nix

```diff
@@ -893,6 +893,9 @@ with pkgs;
 
   gobgp = callPackage ../tools/networking/gobgp { };
 
+  linux-router = callPackage ../tools/networking/linux-router { };
+  linux-router-no-wifi = callPackage ../tools/networking/linux-router { useWifiDependencies = false; };
```

Comment:
@markuskowa Would it be better to use `linux-router-no-wifi = linux-router.override { useWifiDependencies = false; };` here?

Reply:
Yes, using override is the preferred way.

Reply:
Alright, thanks :)